r-package/grf/NAMESPACE

```diff
@@ -1,5 +1,8 @@
 # Generated by roxygen2: do not edit by hand
 
+S3method(leaf_stats,causal_forest)
```

Comment:
Do these `leaf_stats` and `calc_leaf_stats` methods needed to be exported? I don't think we want to expose users to them, as they are only useful to our internal implementation.

Reply:
agreed- I'll remove those exports, thanks!